docs/src/main/paradox/server/index.md

```diff
@@ -0,0 +1,9 @@
+# Providing Services (Server)
+
+@@ toc
+
+@@@ index
+
+ * [Walkthrough](walkthrough.md)
```

Comment:
That's still a bit empty right now. I think we might want to provide more pages with reference information that isn't a walkthrough.